src/realm.h

```diff
@@ -1696,6 +1696,15 @@ RLM_API bool realm_list_get_property(const realm_list_t*, realm_property_info_t*
  */
 RLM_API bool realm_list_get(const realm_list_t*, size_t index, realm_value_t* out_value);
 
+/**
+ * Find the value in the list passed as parameter.
+ * @param value to search in the list
+ * @param out_index the index where the value has been found
```

Comment:
What is this set to if the value hasn't been found?

Reply:
I think we should document the actual value here to make it easier for consumers or alternatively, define it as a constant somewhere so that it can be referenced.

Reply:
I can add a comment, but this is in line with all the other find APIs